sklearn/preprocessing/discretization.py

```diff
@@ -0,0 +1,341 @@
+from __future__ import division
+
+__author__ = "Henry Lin <hlin117@gmail.com>"
```

Comment:
I think our convention is `# Author: Henry Lin <hlin117@gmail.com>`, not sure how much this matters.